.config/cargo_spellcheck.dic

```diff
@@ -0,0 +1,75 @@
+80
+
+WebAssembly
+Wasm
+Polkadot
+validator
+LLVM
+JIT
+rustup
+miri
+clippy
+hunspell
+ERC
+fuzzer
+1A
+2A
+1B
+2B
+SHA2
+BLAKE2
+KECCAK
+ABI
+FFI
+UI
+crypto
+Dev
+deploy
+layed
+dispatchable
+backend
+SRML
+Params
+encodable
+decodable
+spreaded
+callee
+prefilled
+|
```

Comment:
Please show me why you need this "word" in here. Actually we should not include signs but words in this list and see how we can fix those occasions otherwise.